renew-le.sh

```diff
@@ -5,14 +5,14 @@ WORKDIR=$(dirname "$(realpath $0)")
 EMAIL=""
 
 ### cron
-# check that the cert will last at least 2 days from now to prevent too frequent renewal
+# check that the cert will last at least 30 days from now to prevent too frequent renewal
 # comment out this line for the first run
 if [ "${1:-renew}" != "--first-time" ]
 then
-	start_timestamp=`date +%s --date="$(openssl x509 -startdate -noout -in /var/lib/ipa/certs/httpd.crt | cut -d= -f2)"`
+	end_timestamp=`date +%s --date="$(openssl x509 -enddate -noout -in /var/lib/ipa/certs/httpd.crt | cut -d= -f2)"`
 	now_timestamp=`date +%s`
-	let diff=($now_timestamp-$start_timestamp)/86400
-	if [ "$diff" -lt "2" ]; then
+	let diff=($end_timestamp-$now_timestamp)/86400
+	if [ "$diff" -gt "30" ]; then
```

Comment:
Seems like 30 is a long time before expiration. LE sends me a renewal e-mail for my certs 19 days prior to renewal. How they came up with 19 I have no idea. 60 is definitely better than 2, but may be 21 days? Since this is/should be automated it will hopefully not fail too many times. I wonder if someone would notice anyway.

Reply:
that is whats certbot does

https://eff-certbot.readthedocs.io/en/latest/using.html#renewing-certificates
> This command attempts to renew any previously-obtained certificates that expire in less than 30 days.